src/scout_apm/core/core_agent_manager.py

```diff
@@ -9,41 +8,45 @@
 import tarfile
 import time
 
-# 3rd Party
 import requests
 
-# APM Modules
 from scout_apm.core.context import AgentContext
 
-# Logging
 logger = logging.getLogger(__name__)
 
 
-class CoreAgentManager:
+class CoreAgentManager(object):
     def __init__(self):
         self.core_agent_bin_path = None
         self.core_agent_bin_version = None
-        self.core_agent_dir = '{}/{}'.format(AgentContext.instance.config.value('core_agent_dir'),
-                                             AgentContext.instance.config.value('core_agent_full_name'))
-        self.downloader = CoreAgentDownloader(self.core_agent_dir,
-                                              AgentContext.instance.config.value('core_agent_full_name'))
+        self.core_agent_dir = "{}/{}".format(
+            AgentContext.instance.config.value("core_agent_dir"),
+            AgentContext.instance.config.value("core_agent_full_name"),
+        )
+        self.downloader = CoreAgentDownloader(
+            self.core_agent_dir,
+            AgentContext.instance.config.value("core_agent_full_name"),
+        )
 
     def launch(self):
-        if AgentContext.instance.config.value('core_agent_launch') is not True:
-            logger.debug("Not attempting to launch Core Agent "
-                         "due to 'core_agent_launch' setting.")
+        if not AgentContext.instance.config.value("core_agent_launch"):
```

Comment:
I'm not going to mark all truthiness changes - just marking this one again to ensure we discuss them as a whole.